### PR TITLE
Sends cookies associated with a domain in AVURLAsset for tvOS

### PR DESCRIPTION
**Pull request by igoriols**
## Goal

To send cookies associated with a domain when instantiating an `AVURLAsset`.

## How to test

Replace `viewDidLoad()` in `Example/Clapper_tvOS/ViewController.swift` code for this:

```swift
override func viewDidLoad() {
        super.viewDidLoad()

        let cookie = HTTPCookie(properties: [HTTPCookiePropertyKey.domain: "clappr.io",
                                             HTTPCookiePropertyKey.path: "/",
                                             HTTPCookiePropertyKey.name: "testing",
                                             HTTPCookiePropertyKey.value: "it was sent"])!
        HTTPCookieStorage.shared.setCookie(cookie)

        let options = [
            kSourceUrl: "http://clappr.io/highline.mp4",
            kAutoPlay: true,
            kMediaControl: true
            ] as [String: Any]
        player = Player(options: options)

        listenToPlayerEvents()

        addChildViewController(player)
        player.view.frame = view.bounds
        view.addSubview(player.view)
        player.didMove(toParentViewController: self)
    }
```
This code will set a cookie for the domain clappr.io. Then, play the video and check that the cookie was sent (using Charles, for example):

screen shot 2018-05-24 at 14 02 42

### CODE REVIEW

File: Clappr.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (CristianDRM) [inline Clappr.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Should we need to ignore this changes in .gitignore?

File: Sources/Clappr_tvOS/Classes/Plugin/Playback/AVFoundationPlayback/AVFoundationPlayback.swift [hunk 2]
```diff
@@ -151,8 +151,8 @@ open class AVFoundationPlayback: Playback, AVPlayerViewControllerDelegate {
         super.init(options: options)
 
         if let urlString = options[kSourceUrl] as? String {
-            if let url = URL(string: urlString) {
-                asset = AVURLAsset(url: url)
+            if let url = URL(string: urlString), let asset = AVURLAssetWithCookiesBuilder(url: url).asset {
```
Comment (CristianDRM) [inline Sources/Clappr_tvOS/Classes/Plugin/Playback/AVFoundationPlayback/AVFoundationPlayback.swift]: could you extract this code to a new method?

File: Clappr.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 1 above)
Comment (igoriols) [reply Clappr.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Hey @CristianDRM , this file was generated by Xcode 9.3 and had not committed yet.

`Xcode 9.3 adds a new IDEWorkspaceChecks.plist file to a workspace's shared data, to store the state of necessary workspace checks. Committing this file to source control will prevent unnecessary rerunning of those checks for each user opening the workspace.`

[Source](https://developer.apple.com/library/content/releasenotes/DeveloperTools/RN-Xcode/Chapters/Introduction.html#//apple_ref/doc/uid/TP40001051-CH1-DontLinkElementID_7)

File: Sources/Clappr_tvOS/Classes/Plugin/Playback/AVFoundationPlayback/AVFoundationPlayback.swift (hunk 2 above)
Comment (igoriols) [reply Sources/Clappr_tvOS/Classes/Plugin/Playback/AVFoundationPlayback/AVFoundationPlayback.swift]: Done!

**Pull request merged**